samples/DFlow.Samples/Domain/BusinessObjects/Email.cs

```diff
@@ -5,14 +5,11 @@
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
 using System;
-using System.Collections.Generic;
 using DFlow.Domain.BusinessObjects;
-using DFlow.Domain.Validation;
-using DFlow.Samples.BusinessObjects.Domain.BusinessObjects.Validations;
 
 namespace DFlow.Samples.Domain.BusinessObjects
 {
-    public sealed class Email : ValueOf<string, Email, EmailValidator>
+    public sealed class Email : ValueOf<string, Email>
```

Comment:
I am curious about the usage of the string type in ValueOf<**string**

Reply:
It's the programming style of the Value of library. The tekplate have two parameter the primitive type that VO will encapsulate and the VO real type. So string it is the internal type and the second one is the external type.